src/client/components/media-list.vue

```diff
@@ -64,7 +64,7 @@ export default defineComponent({
 
 				if (this.$refs.gridOuter) {
 					let height = 287;
-					const parent = this.$parent.$el;
+					const parent = this.$parent.getElementsByClassName('main');
```

Comment:
```suggestion
					const parent = this.$parent?.$el.getElementsByClassName('main')[0];
```

Reply:
Just tested this change. Works only when you don't change the timeline box width.

Reply:
Well... I messed a bit with the above "commit suggestion".
But what I want to suggest are using then `this.$parent.getElementsByClassName('main')[0];`, works fine how you expect when changes width.